helm/values.yaml

```diff
@@ -13,15 +11,89 @@ sidecars:
   livenessProbeImage:
     repository: quay.io/k8scsi/livenessprobe
     tag: "v2.0.0"
+    pullPolicy: IfNotPresent
   nodeDriverRegistrarImage:
     repository: quay.io/k8scsi/csi-node-driver-registrar
     tag: "v1.3.0"
+    pullPolicy: IfNotPresent
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+priorityClassName: system-node-critical
```

Comment:
now that there is a controller component (the deployment in controller.yaml), probably need to prefix this with `nodePriorityClassName` and also have `controllerPrijorityClassName?